src/jit/importer.cpp

```diff
@@ -2878,24 +2878,36 @@ GenTreePtr      Compiler::impIntrinsic(CORINFO_CLASS_HANDLE     clsHnd,
                                        CORINFO_SIG_INFO *       sig,
                                        int                      memberRef,
                                        bool                     readonlyCall,
-                                       CorInfoIntrinsics *      pIntrinsicID)
+                                       CorInfoIntrinsics *      pIntrinsicID,
+                                       bool *                   pMustExpand)
 {
-    CorInfoIntrinsics intrinsicID = info.compCompHnd->getIntrinsicID(method);
+    CorInfoIntrinsics rawIntrinsicID = info.compCompHnd->getIntrinsicID(method);
```

Comment:
Another option to avoid adding a jit level wrapper and modifying at all places is to add an optional out param pMustExpand to getIntrinsicId().  Then there is no need to extract actual intrinsic id from the raw one.